src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/print_title.html.twig

```diff
@@ -0,0 +1,7 @@
+<span class="h1 print-title">
+  {{ 'Order'|trans({}, 'Admin.Global') }}
+  <span class="print-title__id">
```

Comment:
```suggestion
  <span class="print-title-id">
```

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/17617/commits/880b0ffdaca18f032d2cb941f37e0751b4745479